ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -84,6 +88,9 @@
   private int numChunksTotal;
   // the total number of data chunks retrieved so far (and may or may not have been written out yet).
   private int numChunksRetrieved;
+  // the offset of the first chunk index in the requested byte range
+  // (i.e. 0 if requesting the entire blob or 2 if requesting from the 3rd data chunk on)
+  private int firstChunkOffset = 0;
```

Comment:
That name works too.  I'll change it because it is a bit more consistent with other variable names in this class.